help/destinations/catalog/advertising/magnite-batch.md

```diff
@@ -0,0 +1,190 @@
+| magnite_deviceId_GAID | Google Advertising ID | Select this Target Identity when your source identity is a GAID |
+| magnite_deviceId_IDFA | Apple ID for Advertisers | Select this Target Identity when your source identity is an IDFA |
+| magnite_deviceId_CUSTOM | Custom/user-defined ID | Select this Target Identity when your source identity is not a GAID or IDFA, or if it is a custom or user-defined ID |
+
+{style="table-layout:auto"}
+
+## Supported Audiences {#supported-audiences}
+
+| Audience origin             | Supported | Description | 
+|-----------------------------|----------|----------|
+| [!DNL Segmentation Service] | ✓ | Audiences generated through the Experience Platform [Segmentation Service](../../../segmentation/home.md).|
+| Custom uploads              | ✓ | Audiences [imported](../../../segmentation/ui/overview.md#import-audience) into Experience Platform from CSV files. |
+
+{style="table-layout:auto"}
+
+## Export Type and Frequency {#export-type-frequency}
+
+| Item | Type | Notes | 
+|-----------------------------|----------|----------|
+| Export type | Audience export | You are exporting all members of an audience with the identifiers (name, phone number, or others) used in the Magnite Streaming: Batch destination. |
+| Export frequency | Batch | Batch destinations export files to downstream platforms in increments of three, six, eight, twelve, or twenty-four hours. Read more about batch [file-based destinations](../../../destinations/destination-types#file-based). |
+
+{style="table-layout:auto"}
+
+## Connect to the destination {#connect}
+
+Once your destination usage has been approved and Magnite Streaming has shared your credentials, please follow the below steps to authenticate, map, and share data.
+
+### Authenticate to destination {#authenticate}
+
+Locate the Magnite Streaming: Batch destination in the Adobe Experience catalog. Click the additional options button (\...) and then configure the destination connection/instance.
+
+If you already have an existing account, you can locate it by changing the Account type option to "Existing account". Otherwise, you will create an account below:
+
+To create a new account, and authenticate it to the destination for the first time, fill in the required "S3 access key" and "S3 secret key" fields (provided to you via your account manager), and select **[!UICONTROL Connect to destination]**
+
+![destination configuration auth fields unfilled](../../assets/catalog/advertising/magnite/destination-batch-config-auth-unfilled.png)
+
+**Note:** Magnite Streaming's security policy requires a regular rotation of S3 keys. You should expect to update your account in the future with new S3 access and S3 secret keys. You only need to update the account itself - destinations using that account will automatically use the updated keys. Failure to upload the new keys will result in the data failing to send to this destination.
```

Comment:
This sounds like an important callout.

First, please use the !NOTE formatting as shown in the authoring guide: https://experienceleague.adobe.com/en/docs/contributor/contributor-guide/writing-essentials/markdown#note-blocks.

Second, let's also add a link to this page, which describes how users can update an account: https://experienceleague.adobe.com/en/docs/experience-platform/destinations/ui/update-accounts